cwd/app.js

```diff
@@ -313,13 +318,7 @@
    */
 
   function startKiosk(duration) {
-    views = allGlyphs.filter(obj => obj.view).map(obj => obj.view);
     let index = 0;
```

Comment:
Given `index` is used in the switching code, should it also be moved above as the other variables were?

Reply:
Yeah you're probably right that that'd be better organizationally.